test/easylistTestCases.js

```diff
@@ -0,0 +1,60 @@
+const easylistTestCases = [
+    // blocking by address parts
+    // /banner/*/img
+    {'url': 'http://example/banner/foo/img', 'block': true, 'options': {}},
+    {'url': 'http://example/banner/foo/bar/img?param', 'block': true, 'options': {}},
+    {'url': 'http://example/banner//img/foo', 'block': true, 'options': {}},
+    {'url': 'http://example/banner/img', 'block': false, 'options': {}},
+    {'url': 'http://example/banner/foo/imgraph', 'block': false, 'options': {}},
+    {'url': 'http://example/banner/foo/img.gif', 'block': false, 'options': {}},
+
+    // blocking by domain
+    // ||ads.example.com^
+    {'url': 'http://ads.example.com/foo.gif', 'block': true, 'options': {}},
+    {'url': 'http://server1.ads.example.com/foo.gif', 'block': true, 'options': {}},
+    {'url': 'http://ads.example.com:8000/', 'block': true, 'options': {}},
+    {'url': 'http://ads.example.com.ua/foo.gif', 'block': false, 'options': {}},
+    {'url': 'http://example2.com/foo.gif', 'block': false, 'options': {}},
+    {'url': 'http://example.info/redirect/http://example.com/', 'block': false, 'options': {}},
```

Comment:
Want to add an additional a test case here for:
`http://example.com/redirect/http://ads.example.com/`

Reply:
Done

Reply:
Haha wait I thumbsed up but don't see additional rule. Was asking for additional test for: 
`http://example.com/redirect/http://ads.example.com/`

Reply:
(The additional code comment is nice tho)

Reply:
This one? https://github.com/duckduckgo/chrome-zeroclickinfo/pull/226/files#diff-5220b32111039a4ab4eedd1226bc17a4R38

Reply:
Yes, thanks, sorry github displayed your addition beneath the comment here. Merging.